.semaphore/vulnix-nightly.yml

```diff
@@ -0,0 +1,65 @@
+---
+version: "v1.0"
+name: "Icepeak CI Pipeline"
+
+agent:
+  machine:
+    type: "e1-standard-2"
+    os_image: "ubuntu2004"
+
+global_job_config:
+  env_vars:
+    - name: "CRONJOB_HEALTHCHECK_URL"
+      value: "https://hc-ping.com/d80327c2-4cbc-4986-b4bb-c43002938f00"
```

Comment:
Because this repository is public I'd prefer to not have this healthcheck URL visible. Please delete and regenerate the healthcheck, then we can add it as a secret in Semaphore.